apps/test/unit/templates/sectionProgressV2/LevelTypesBoxTest.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import {expect} from '../../../util/reconfiguredChai';
+import LevelTypesBox from '@cdo/apps/templates/sectionProgressV2/LevelTypesBox';
+
+describe('LevelTypesBox Component', () => {
+    it('renders three options', () => {
+      render(<LevelTypesBox />);
+      expect(screen.getByText('Level Types')).to.exist;
```

Comment:
Side note, in all of these tests I rely more on testing that the text shows up rather than the symbol.  I am interested in knowing if that feels sufficient, given I test the symbols show up in the test of `ProgressIcon`